### DISCUSSION TITLE
Get extrinsic and intrinsic parameters for a camera view

### DISCUSSION THREAD

**Discussion by amilasilva92** [pyvista/pyvista#2947] [Q&A]
Hello, 

I have the following problem at hand:
I have a 2d camera view of a 3d object that I captured using PyVista. Now, I have a set of 3d points and I want to find the exact 2d locations for those points from the same camera view on the previously compiled image. I am thinking to extract the extrinsic and intrinsic parameters for the camera view (as the following format) and do the projection for the 3d points. 

```
intrinsic = [f 0 cx
             0 f cy
             0 0 1]

extrinsic = [R t]
```

However, I was unable to find a direct method in PyVista to extract the camera parameters. I would really appreciate if someone can help me to solve this task. Thank you.

**Comment by YiftachEde** [comment]
Have you been successfull in extracting those parameters from the camera view?

**Reply by amilasilva92** [reply]
Hey, I was unable to find a solution.

**Comment by ZYX-MLer** [comment]
I encountered the same problem.
